layouts/millennium/r4/financial/support.html

```diff
@@ -1,3 +1,3 @@
-<p>Financial Support resources include the Coverage, CoverageEligibilityRequest, CoverageEligibilityResponse, EnrollmentRequest, and EnrollmentResponse resources.</p>
+<p>Financial Support resources include the Coverage, CoverageEligibilityRequest, CoverageEligibilityResponse, EnrollmentRequest, and EnrollmentResponse resources, and InsurancePlan</p>
```

Comment:
Need to put it inside the financial/general as per Bhavya's comment

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/809/commits/f285387f72f531a780abd964a8c1b3335e9d5385#diff-1b2e7eaf4e7886cd7feb2c3259aaf69b731e611ada9084e5910d2e63deff97ddR13